debian/changelog

```diff
@@ -1,3 +1,9 @@
+revpi-repo (3.0.0-1+revpi11+1) bullseye; urgency=medium
+
+  * Update repositories to Bullseye
```

Comment:
https://github.com/RevolutionPi/revpi-repo/commit/ef61cf6d5c90c7e30d6c82d8bf7f24fdf208ae5a
gbp dch should prevent this ;)
```suggestion
  *  Update package list to Bullseye
```